GoogleForCreators/web-stories-wp
# Reset zoom to fit when starting drag or inserting element from library

## Context

This resets the zoom to fit regardless of what it was when inserting an element from the library by clicking or dragging. This works for all elements including:

* Local media
* Third-party media
* Text presets and text sets
* Shapes

## User-facing changes

| Before | After |
|-|-|
| ![click-local-before](https://user-images.githubusercontent.com/637548/117226389-917c7900-ade2-11eb-8122-146e4fa10426.gif) | ![click-local-after](https://user-images.githubusercontent.com/637548/117226396-93ded300-ade2-11eb-8169-fe1f768fd05b.gif) |
| ![drag-local-before](https://user-images.githubusercontent.com/637548/117226401-95a89680-ade2-11eb-9d84-034e014ff80e.gif) | ![drag-local-after](https://user-images.githubusercontent.com/637548/117226404-96d9c380-ade2-11eb-8416-23423f09cdd5.gif) |
| ![click-text-before](https://user-images.githubusercontent.com/637548/117226407-98a38700-ade2-11eb-8a51-abfe673c07f1.gif) | ![click-text-after](https://user-images.githubusercontent.com/637548/117226408-99d4b400-ade2-11eb-9be8-eb51436b4e4d.gif) |
| ![click-shape-before](https://user-images.githubusercontent.com/637548/117226413-9b05e100-ade2-11eb-876d-f33d1a3d5ec0.gif) | ![click-shape-after](https://user-images.githubusercontent.com/637548/117226415-9c370e00-ade2-11eb-9105-008d76c49c46.gif) |

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Set the canvas zoom to anything but fit
1. Insert any element from library by clicking or dragging
1. Observe the zoom reset to fit before the element is inserted / before dragging starts

### UAT




## Checklist




---

Fixes #6182
